### ISSUE TITLE
Editor SceneBar has useless buttons for certain categories

### ISSUE THREAD

**Issue by trundlr** [Facepunch/sbox-issues#6801]
### Describe the bug

Categories are titled but some have useless "buttons" that are seemingly supposed to be pseudo-titles but don't do anything:

![Image](https://github.com/user-attachments/assets/4bfaf6b4-92d8-42a5-8ec1-da747d47f694)

Navigation mesh has the "Navigation" icon which is redundant and Network has the "Multiplayer" icon which is redundant.

### To Reproduce

-

### Expected behavior

-

**Comment by Nolankicks** [comment]
I agree that this should be fixed, but I think the top bar is a larger issue. It's very cluttered with tons of buttons that are hard to understand for some people. I regularly find myself explain to someone over call which buttons they should press.
